app/build_wasm-mac.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
btw, I'm quite tempted to remove the webassembly part entirely from WPD at some point. I added the support hoping to be able to leverage libraries like Eigen, but I'm not very certain if that was a good move.